src/pages/Swap/index.tsx

```diff
@@ -644,6 +643,10 @@ export default function Swap() {
               isStableSwapPriceImpactSevere={isStableSwapPriceImpactSevere}
             />
           )}
+          <StyledAddToMetaMaskButton
+            token={currencies.OUTPUT as Token}
+            swapDetailsOpen={!!trade || !!stableswapTrade}
```

Comment:
```suggestion
            swapDetailsOpen={trade != null || stableswapTrade != null}
```

Let's avoid type coercion if we can